packages/components/src/components/toast/toast.tsx

```diff
@@ -3,7 +3,24 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { CSSTransition } from 'react-transition-group';
 
-const Toast = ({ children, className, is_open = true, onClose, onClick, type = 'info', timeout = 0 }) => {
+type TToast = {
+    className?: string;
+    is_open?: boolean;
+    onClick: (event: React.MouseEvent<HTMLElement>) => void;
+    onClose: () => void;
+    type: 'error' | 'info';
+    timeout: number;
```

Comment:
```suggestion
    onClick?: React.MouseEventHandler<HTMLDivElement>;
    onClose?: () => void;
    type?: 'error' | 'info';
    timeout?: number;
```
@likhith-deriv as I can see from the component usage, all props are non-required🙂